pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -176,6 +178,8 @@ async def serve_requests(self, num_requests=None):
                     result = self._call_drain_handler()
                     if asyncio.iscoroutine(result):
                         await result
+                    self._discard_events = True
```

Comment:
@quaark The event cannot sneak anywhere, because we process events in the same loop in which we wait for the result, so I believe there are no difference